.github/workflows/build-test-core-win64.yml

```diff
@@ -0,0 +1,81 @@
+# AUTOGENERATED FROM build-test-core-win64.jsonnet DO NOT MODIFY
+name: build-test-core-win64
+"on":
+  workflow_dispatch:
+  workflow_call:
+jobs:
+  job:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash  # Windows GHA runners default to pwsh (PowerShell). We want to use bash to be consistent with our other workflows.
+    steps:
+    - name: Configure git
+      run: |
+        # Git's filename limit is 4096 characters, except on Windows when Git is compiled with msys.
+        # It uses an older version of the Windows API and there's a limit of 260 characters for a filename.
+        # You can force the longer limit (and avoid git issues) by enabling core.longpaths in your git config.
+        # More info: https://stackoverflow.com/a/22575737
+        git config --system core.longpaths true
+
+        # Speed up checkout by running multiple fetches in parallel
+        git config --global fetch.parallel 50
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: "4.14"
+        opam-pin: false
```

Comment:
would be good to comment those settings.
both the pin and the allow-prelease-opan

Reply:
I'm actually going to remove the prerelease option -- there's no prereleases for windows (it's using the fdopen repo AFAIK). Good call on documenting opam-pin.

Reply:
Actually I can probably remove `opam-pin` too now that I'm pointing the action at a bogus opam package...